templates/header/global_alert.html

```diff
@@ -5,7 +5,7 @@
 {%- if global_alert -%}
     <li class="pure-menu-item">
         <a href="{{ global_alert.url | safe }}" class="pure-menu-link {{ global_alert.css_class }}">
-            <i class="{{ global_alert.fa_icon | fas(fw=true) }}"></i>
+            <svg class=fa-svg viewBox="0 0 512 512"><use xlink:href="/-/static/fa-solid.svg#{{ global_alert.fa_icon }}"></svg>
```

Comment:
Additionally I don't think this link will work since you removed the routing to the files that were served

Reply:
Oh, right. I missed that one.